### PR TITLE
Update privacy policy, Add GDPR information.

### PR DESCRIPTION
**Pull request by ryanstrat**
Here's what it looks like rendered: https://app.usebubbles.com/FrWbm7BCq7PrcoRpShkzv/comments-on-my-robojackets-org/

### CODE REVIEW

File: resources/views/privacy.blade.php [hunk 1]
```diff
@@ -13,29 +13,73 @@
...
             </ul>
-            <p>RoboJackets will only release information about you as strictly necessary to Georgia Tech campus departments performing work on our behalf, including but not limited to granting BuzzCard access to the shop, enrollment in SUMS, and reimbursement processing. RoboJackets will release the resumes of all active members to our sponsors.</p>
-            <p>Demographic information may be reported in aggregate to current or potential sponsors.</p>
-            <p>All other information will be released only to RoboJackets officers in the course of conducting business.</p>
+            <p>RoboJackets does not sell any of the information collected to any organizations. However, sponsors who provide financial support to RoboJackets will receive aggregated data as outlined above as a benefit of their support.</p>
+
+            <h2>Cookies</h2>
+            <p>This application uses cookies to keep you logged in during and between sessions. If you elect to disable first-party cookies in your browser settings, this website may no longer function as intended.</p>
```
Comment (JoshOldenburg) [inline resources/views/privacy.blade.php:70]: Add commas (in, during, and between)

File: resources/views/privacy.blade.php [hunk 2]
```diff
@@ -13,29 +13,73 @@
...
                 <li>Your personal email address</li>
                 <li>Your preferred name</li>
                 <li>Your phone number</li>
                 <li>Your emergency contact information, including their name and phone number</li>
                 <li>Your shirt and polo sizes</li>
-                <li>Your gender</li>
-                <li>Your ethnicity</li>
                 <li>Your resume</li>
+                <li>The usernames or other identifiers of your linked Github, Google, and ClickUp accounts</li>
+                <li>Transaction identifiers for credit/debit card transactions completed by RoboJackets' payments processor, Square</li>
```
Comment (JoshOldenburg) [inline resources/views/privacy.blade.php:37]: Clarify that we do not handle credit card numbers

File: resources/views/privacy.blade.php [hunk 3]
```diff
@@ -13,29 +13,73 @@
...
+            <h2>What data does RoboJackets collect?</h2>
+            <p>RoboJackets stores the following information about you each time you log in, as reported by the Georgia Tech Enterprise Directory. You may opt out of this data collection by not using the application.</p>
             <ul>
                 <li>Your first name</li>
                 <li>Your last name</li>
                 <li>Your primary Georgia Tech email address</li>
                 <li>Your GTID number</li>
                 <li>Your Georgia Tech username</li>
+                <li>Your major</li>
+                <li>Your class standing</li>
```
Comment (JoshOldenburg) [inline resources/views/privacy.blade.php:26]: This list should include nginx IP logs (I need to check what exactly we log)

**Review by JoshOldenburg — CHANGES_REQUESTED** [review]
A few minor changes. Overall it's a big improvement.

File: resources/views/privacy.blade.php (hunk 1 above)
Comment (ryanstrat) [reply resources/views/privacy.blade.php:70]: It's not a list. We could hyphenate logged-in to improve clarity, perhaps.

File: resources/views/privacy.blade.php (hunk 3 above)
Comment (ryanstrat) [reply resources/views/privacy.blade.php:26]: That's technically explicitly covered by GT's policy, which we extend. But we can add it in here as well.

File: resources/views/privacy.blade.php (hunk 1 above)
Comment (JoshOldenburg) [reply resources/views/privacy.blade.php:70]: A hyphen would be good. I misread that twice.

**Review by JoshOldenburg — DISMISSED** [review]

File: resources/views/privacy.blade.php [hunk 4]
```diff
@@ -13,29 +13,74 @@
...
+                <li>Transaction identifiers for credit/debit card transactions completed by RoboJackets' payments processor, Square; RoboJackets does not process or store billing data, including credit/debit card numbers.</li>
+                <li>A history of events and team meetings attended</li>
+            </ul>
+
+            <h2>How will RoboJackets use your data?</h2>
+            <ul>
+                <li>To track membership activity, including dues transactions and meeting attendance</li>
+                <li>To facilitate all payments via credit and debit cards</li>
+                <li>To purchase and distribute items included with membership, such as t-shirts and polos</li>
+                <li>To provision and manage access to computer systems that RoboJackets uses or maintains, such as Google Drive, RoboJackets Cloud, Github, ClickUp, SUMS, and Wordpress</li>
```
Comment (kberzinch) [inline resources/views/privacy.blade.php:47]:
```suggestion
                <li>To provision and manage access to computer systems that RoboJackets uses or maintains, such as Google Drive, RoboJackets Cloud, GitHub, ClickUp, SUMS, and WordPress</li>
```

**Review by kberzinch — CHANGES_REQUESTED** [review]

**Review by kberzinch — APPROVED** [review]

**Pull request merged**